src/watch/index.ts

```diff
@@ -19,20 +19,18 @@ export interface WatcherOptions {
 	clearScreen?: boolean;
 }
 
-export type InputAndOutputOptions = InputOptions & OutputOptions;
-
-export interface RollupWatchOptions extends InputAndOutputOptions {
+export interface RollupWatchOptions extends InputOptions, OutputOptions {
```

Comment:
in all fairness, I am not sure if `RollupWatchOptions` should extends `InputOptions, OutputOptions`

@lukastaegert can you confirm that this is correct and it should be 
```ts
export interface RollupWatchOptions  {
	output?: OutputOptions;
	watch?: WatcherOptions;
}
```

Reply:
As far as I see it, it should probably look like

```ts
export interface RollupWatchOptions extends InputOptions {
	output?: OutputOptions | OutputOptions[];
	watch?: WatcherOptions;
}
```
I.e. `RollupWatchOptions` should look like legal rollup options. The main problem is that there are a lot of wrong types floating around in the config handling.  Cf. https://github.com/rollup/rollup/pull/1837#issuecomment-356523806